src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -240,10 +241,10 @@ def register_extra_cli_argument(command, dest, **kwargs):
     _cli_extra_argument_registry[command][dest] = CliCommandArgument(dest, **kwargs)
 
 def cli_command(module_name, name, operation,
```

Comment:
Just so long as the public facing `cli_command(...)` call does not use kwargs (*cough*resource_id*cough*)